Mathlib/CategoryTheory/Functor/FullyFaithful.lean

```diff
@@ -212,6 +212,13 @@ def isoEquiv {X Y : C} : (X ≅ Y) ≃ (F.obj X ≅ F.obj Y) where
 def comp {G : D ⥤ E} (hG : G.FullyFaithful) : (F ⋙ G).FullyFaithful where
   preimage f := hF.preimage (hG.preimage f)
 
+/-- Given a natural isomorphism between `F ⋙ H` and `G ⋙ H` for a fully faithful functor `H`, we
+can 'cancel' it to give a natural iso between `F` and `G`. -/
+def cancelRight {C D E : Type*} [Category C] [Category D] [Category E] {F G : C ⥤ D} {H : D ⥤ E}
+    (HH : H.FullyFaithful) (comp_iso : F ⋙ H ≅ G ⋙ H) : F ≅ G :=
+  NatIso.ofComponents (fun X ↦ HH.preimageIso (comp_iso.app X)) fun f ↦
```

Comment:
This is a particular case of `(HH.whiskeringRight C).preimageIso comp_iso` from `CategoryTheory.Whiskering`.

Reply:
Do you think we should have this special case explicitly or not?

Reply:
I do not think we need this.